recipes/argp-standalone/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "argp" %}
+{% set version = "1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+    url: https://www.lysator.liu.se/~nisse/misc/argp-standalone-{{ version }}.tar.gz
+    sha256: dec79694da1319acd2238ce95df57f3680fea2482096e483323fddf3d818d8be
+
+build:
+  number: 0
+  skip: true  # [win or linux]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x' ) }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make
+
+test:
+  commands:
+    - test -f ${PREFIX}/lib/libargp.a
+    - test -f ${PREFIX}/include/argp.h
+
+about:
+    home: https://www.lysator.liu.se/~nisse/misc
+    license: GPL-2.0-or-later
```

Comment:
you included a license file for BSD3?

Reply:
Yes I think I was confused when looking for packages with a GPLv2 license, the feedstock is BSD3.